File: Treino/Semana 1/Daniel/contagem.py
appeared_number = [0 for i in range(10)]
N = int(input())

for i in range(N):
    number = input()
    for digit in number:
        appeared_number[int(digit)] += 1

for i in range(10):
    print(f'{i} - {appeared_number[i]}')

File: Treino/Semana 1/contagem.cpp
#include <iostream>

using namespace std;

int N;
int appearedNumber[] = {0, 0, 0, 0, 0, 0, 0, 0, 0, 0};

int main()
{
    cin >> N;

    for (int i = 0; i < N; i++)
    {
        string number;
        cin >> number;
        for (auto digit: number)
        {
            int numberInt;
            numberInt = digit - '0';
            appearedNumber[numberInt]++;
        }
    }

    for (int i = 0; i < 10; i++) cout << i << " - " << appearedNumber[i] << endl;
    return 0;
}